include/swift/SILOptimizer/Analysis/IVAnalysis.h

```diff
@@ -26,7 +26,6 @@ namespace swift {
 class IVInfo : public SCCVisitor<IVInfo> {
 public:
   typedef llvm::SmallVectorImpl<ValueBase *> SCCType;
-  friend class SCCVisitor;
```

Comment:
I think the right way to fix this is just `friend SCCVisitor;`, not to make the `visit` method public.